src/app/devices/deviceContent/components/digitalTwin/digitalTwinInterfaces.tsx

```diff
@@ -22,7 +22,7 @@ import '../../../../css/_devicePnPDetailList.scss';
 
 export interface DigitalTwinInterfacesProps extends RouteComponentProps{
     isLoading: boolean;
-    idToNameMap: Map<string, string>;
+    nameToIds: object;
```

Comment:
What is the data structure here? Is there anything better than object that we can us?

Reply:
Please see the comments below.